tensorflow/lite/micro/python/interpreter/src/tflm_runtime.py

```diff
@@ -39,16 +35,14 @@ def __init__(self,
       arena_size = len(model_data) * 10
 
     # Some models make use of resource variables ops, get the count here
-    if num_resource_variables != 0:
-      print(
-          'WARNING: num_resource_variables is deprecated. Resource variable count is ',
-          'calculated automatically.')
-    number_resource_variables = flatbuffer_utils.count_resource_variables(
+    num_resource_variables = flatbuffer_utils.count_resource_variables(
         model_data)
+    if num_resource_variables != 0:
+      print("Note: Model makes use of resource variables, count = ",
+            num_resource_variables)
```

Comment:
let's print even if the number is zero. So there is no ambiguity.

Reply:
Done.